py-polars/src/conversion/any_value.rs

```diff
@@ -371,51 +390,50 @@ pub(crate) fn py_object_to_any_value(ob: &PyAny, strict: bool) -> PyResult<AnyVa
             match &*type_name {
                 // Can't use pyo3::types::PyDateTime with abi3-py37 feature,
                 // so need this workaround instead of `isinstance(ob, datetime)`.
-                "date" => get_date,
-                "time" => get_time,
-                "datetime" => get_datetime,
-                "timedelta" => get_timedelta,
-                "Decimal" => get_decimal,
-                "range" => get_list,
+                "date" => get_date as InitFn,
```

Comment:
Lacking the `as InitFn` I got errors about "expected fn pointer, got fn item" (or vice versa?).

Reply:
(Unresolving temporarily, just in case someone else reviews)